gc/base/ParallelDispatcher.cpp

```diff
@@ -457,6 +457,26 @@ MM_ParallelDispatcher::recomputeActiveThreadCountForTask(MM_EnvironmentBase *env
 	 * available and ready to run).
 	 */
 	uintptr_t taskActiveThreadCount = OMR_MIN(_activeThreadCount, threadCount);
+
+	/* Account for Adaptive Threading. Ignore if
+	 *
+	 *  1) User forced a thread count (e.g Xgcthreads)
+	 *  2) Adaptive threading flag is not set (-XX:-AdaptiveGCThreading)
+	 *  3) The task wasn't recommended a thread count (currently only recommended for STW Scavenge Tasks)
+	 */
+	if (_extensions->adaptiveThreadigEnabled() && task->getRecommendedWorkingThreads() != UDATA_MAX) {
+		/* Bound the recommended thread count. Determine the  upper bound for the thread count,
+		 * either the user specified adaptiveThreadCount (-XXgc:adaptiveThreads=) or else default max
```

Comment:
I think we should make the option -Xgc: (single X, documented, supported)

Reply:
Changed - see https://github.com/eclipse/openj9/pull/11999